From a5b7b1194a57bc59f289f3e4433a1be81cc3e19d Mon Sep 17 00:00:00 2001
Message-Id: <a5b7b1194a57bc59f289f3e4433a1be81cc3e19d.1649664492.git.geliang.tang@suse.com>
From: Veronika Kabatova <vkabatov@redhat.com>
Date: Thu, 10 Dec 2020 13:01:34 +0100
Subject: [PATCH] selftests/bpf: Drop tcp-{client,server}.py from Makefile
Git-commit: a5b7b1194a57bc59f289f3e4433a1be81cc3e19d
Patch-mainline: v5.11-rc1
References: bsn#127
Modified-by-SEL: No


The files don't exist anymore so this breaks generic kselftest builds
when using "make install" or "make gen_tar".

Fixes: 247f0ec361b7 ("selftests/bpf: Drop python client/server in favor of threads")
Signed-off-by: Veronika Kabatova <vkabatov@redhat.com>
Signed-off-by: Daniel Borkmann <daniel@iogearbox.net>
Link: https://lore.kernel.org/bpf/20201210120134.2148482-1-vkabatov@redhat.com
Signed-off-by: Geliang Tang <geliang.tang@suse.com>
---
 tools/testing/selftests/bpf/Makefile | 2 --
 1 file changed, 2 deletions(-)

diff --git a/tools/testing/selftests/bpf/Makefile b/tools/testing/selftests/bpf/Makefile
index 944ae17a39ed..50b3495d7ddf 100644
--- a/tools/testing/selftests/bpf/Makefile
+++ b/tools/testing/selftests/bpf/Makefile
@@ -75,8 +75,6 @@ TEST_PROGS := test_kmod.sh \
 
 TEST_PROGS_EXTENDED := with_addr.sh \
 	with_tunnels.sh \
-	tcp_client.py \
-	tcp_server.py \
 	test_xdp_vlan.sh
 
 # Compile but not part of 'make run_tests'
-- 
2.34.1

